test/test_tseries.jl

```diff
@@ -29,14 +29,14 @@ import TimeSeriesEcon: qq, mm, yy
     @test s[12] == 22.0
     @test s[1:3] == [11.0, 12.0, 13.0]
     @test s[1:2:12] == collect(10.0 .+ (1:2:12))
-    @test s[s .< 13] == [11.0, 12.0]
+    @test s[s.<13] == [11.0, 12.0]
```

Comment:
I know we haven't really discussed linters, but this linting rule seems bad for readability.